package.json

```diff
@@ -48,6 +48,7 @@
   },
   "dependencies": {
     "@ant-design/icons": "5.0.1",
+    "@dnd-kit/core": "^6.0.8",
```

Comment:
How does this library compare to alternatives? What is its footprint in terms of bundle size?

Reply:
I did a bit of research, before choosing it.

https://npmtrends.com/@dnd-kit/core-vs-react-beautiful-dnd-vs-react-dnd-vs-react-drag-and-drop-vs-react-draggable

> the core of the library weighs around 10kb minified

> It is a zero-dependent, feature-rich dnd library

> Dnd-kit is a pretty good lightweight solution

Let me know if you have another recommendation.

Reply:
Looks good, thanks.